src/App.css

```diff
@@ -13,20 +13,21 @@
   }
 }
 
+
+
 .App-header {
-  background-color: white; /* #282c34 */
+  background-color: #282c34;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
-  justify-content: flex-start;
+  align-items: center;
+  justify-content: center;
   font-size: calc(10px + 2vmin);
```

Comment:
Font sizes will be constant - they can be imported from CSS variables as well

Reply:
Ok